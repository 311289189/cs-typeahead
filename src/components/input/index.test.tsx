import React from 'react';
import { render } from '@testing-library/react';
import Input from './';

test('renders <Input/> component', () => {
    const { container } = render(<Input />);
    const inputElement = container.querySelector('input')
    expect(inputElement).toBeInTheDocument();
});
