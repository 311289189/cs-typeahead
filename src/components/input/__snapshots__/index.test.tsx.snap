// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<input
  aria-label="cost-input"
  class="generic-input"
  type="text"
/>
`;
